packages/terra-overlay/translations/de.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Reached out to Kelsey. She gave me the link to the Translation's team's Jira queue and she sent me the translations for French, German, Portuguese, Spanish & Uk English.